scripts/docker/build-image-ubi9.sh

```diff
@@ -19,3 +20,8 @@ docker build \
   --build-arg NODE_16_LATEST_VERSION_TAR_GZ_FILE_SHASUM256="${NODE_16_LATEST_VERSION_TAR_GZ_FILE_SHASUM256}" \
   -t ${NAME_AND_TAG} \
   --file=Dockerfile.ubi9 .
+
+# Smoke test image binaries
+docker run --entrypoint /usr/bin/dumb-init ${NAME_AND_TAG} --version
```

Comment:
Nice! What about adding some extra flags to ensure:
* it doesn't hang for a long time (e.g, `docker run [...] timeout 20`)
* the container is removed upon exit (e.g `docker run --rm [...]`)

Reply:
`dumb-init` is in control so the timeout won't be able to function anyway if it's broken, and if it's working then it'll exit quick. It would be nice to have some way of stopping it hanging though 🤔 `--rm` doesn't matter in CI but is good locally so I'll add it.

Reply:
I've added a bash timeout to the whole `docker run` command.